evm/vm/forks/sharding/opcodes.py

```diff
@@ -18,10 +20,18 @@
         mnemonic=mnemonics.SIGHASH,
         gas_cost=0,
     ),
+    opcode_values.CREATE2: system.Create2.configure(
+        name='opcode:CREATE2',
+        mnemonic=mnemonics.CREATE2,
+        gas_cost=constants.GAS_CREATE2,
+    )(),
 }
 
 
 SHARDING_OPCODES = merge(
     copy.deepcopy(BYZANTIUM_OPCODES),
     NEW_OPCODES
 )
+
+
+SHARDING_OPCODES.pop(opcode_values.CREATE)
```

Comment:
Instead of popping this value out, can we do the following a few lines up.

```python
from cytoolz import dissoc

SHARDING_OPCODES = merge(
     dissoc(copy.deepcopy(BYZANTIUM_OPCODES), opcode_values.CREATE),
     NEW_OPCODES
 )
```